modules/calendar/spec/features/calendar_user_interaction_spec.rb

```diff
@@ -34,17 +34,16 @@
                :selenium do
   include_context "with calendar full access"
 
-  let!(:other_user) do
-    create(:user,
-           firstname: "Bernd",
-           member_with_permissions: { project => %w[view_work_packages view_calendar] })
-  end
-
-  let!(:work_package) do
+  shared_let(:monday) { Time.zone.today.beginning_of_week }
```

Comment:
This is only true, when `Date.beginning_of_week` is set to `:monday`. This might not be true on all systems, to ensure that we always have the monday, let's use:

```suggestion
  shared_let(:monday) { Time.zone.today.beginning_of_week(:monday) }
```

Reply:
Sweet! I didn't know `beginning_fo_week` accepted parameters. (Well, looking at [some of my commits](https://github.com/opf/openproject/commit/3b20568377cd097776923c90f99605848ef8e580#diff-b51c5cf8597049c3bdf710622f5fbcabaa166b99ce40c906e552e7f81eaa34bcR37), it looks like I knew it, and forgot it)
Very good suggestion, I'll add it.